packages/ui-scripts/package.json

```diff
@@ -18,7 +18,7 @@
   "scripts": {
     "lint": "ui-test --lint",
     "lint:fix": "ui-test --lint --fix",
-    "build:watch": "tsc -w -p tsconfig.build.json",
+    "build:watch": "tsc  -b tsconfig.build.json -w",
```

Comment:
This might crash with Matyi's change (he merged it yesterday)

Reply:
Should be fixed